src/datasources/cache/redis.cache.service.spec.ts

```diff
@@ -153,7 +170,7 @@ describe('RedisCacheService', () => {
     expect(redisClientTypeMock.quit).toHaveBeenCalledTimes(0);
   });
 
-  it('Deleting key calls delete', async () => {
+  it("Deleting key calls delete and don't set invalidationTime", async () => {
```

Comment:
I understand the need for this test but this situation only happens because the interface that is offered allows for disabling this behaviour (although actual usage seems to be limited to having the `invalidationTime` always registered).

Reply:
Removed in [b92584c](https://github.com/safe-global/safe-client-gateway/pull/970/commits/b92584cf5c50470b34305c8c9afffcc884c9fb21)